pysipp/xml/minidom.py

```diff
@@ -0,0 +1,80 @@
+import locale
+import functools
+from xml.dom.minidom import *
+from xml.dom.minidom import _write_data
+
+
+@functools.total_ordering
+class AttributeSorter(object):
+    def __init__(self, obj, *args):
+        self.obj = obj
+
+    def __lt__(self, other):
+        if self.obj in ('request', 'response'):
+            return True
+        elif other.obj in ('request', 'response'):
+            return False
+        return locale.strcoll(self.obj, other.obj) < 0
+
+
+class Newline(CharacterData):
+    __slots__ = ()
+
+    nodeType = Node.TEXT_NODE
+    nodeName = "#text"
+    attributes = None
+
+    def writexml(self, writer, indent="", addindent="", newl=""):
+        writer.write(newl)
+
+
+def createSeperator(self):
+    c = Newline()
+    c.ownerDocument = self
+    return c
+
+
+Document.createSeperator = createSeperator
```

Comment:
There's "a-rat" in separator ;)

(That 'a' is audible in romanic/germanic languages, causing the typo to stand out.)